src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -25,6 +25,15 @@ import {CDK_MENU, Menu} from './menu-interface';
 import {CdkMenuItem} from './menu-item';
 import {MenuStack, MenuStackItem, FocusNext} from './menu-stack';
 
+/**
+ * Check if the given element is part of the cdk menu module.
+ * @param target the element to check.
+ * @return true if the given element is part of the menu module.
+ */
+function isMenuElement(target: Element) {
```

Comment:
The name is a little ambiguous. Maybe a more verbose name, 'isElementAMenu', would be clearer?

Reply:
@jelbourn thoughts?

Reply:
I'd prefer `isMenuElement`